### PR TITLE
add groups.io

### PR DESCRIPTION
**Pull request by rajiv**
[groups.io](https://groups.io) alexa rank [34,376](https://www.alexa.com/siteinfo/groups.io)

### CODE REVIEW

**Comment by tespups** [comment]
Help catch me up

**Comment by Carlgo11** [comment]
Thanks for the pull request!

I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
screenshot

**Comment by rajiv** [comment]
there is no public documentation. 

once logged in, a user can go to https://groups.io/account?page=security to configure 2fa. once enabled, the page looks like this:


Account

**Comment by phallobst** [comment]
This seems to fit better into the "Communication" category instead of "Email".

**Comment by Carlgo11** [comment]
@Rajiv you don’t need to request me to review your PR. Michael already told you what to fix.

**Review by phallobst — APPROVED** [review]

File: _data/communication.yml [hunk 1]
```diff
@@ -129,6 +129,12 @@ websites:
       - sms
     doc: https://help.groupme.com/hc/en-us/articles/360000579167
 
+  - name: Groups.io
+    url: https://groups.io
+    img: groups_io.png
+    tfa:
+      - totp
+
```
Comment (Jawshy) [inline _data/communication.yml:147]:
```suggestion
doc: "/notes/groups.io/"

```
As there is no documentation by groups.io for setting up 2FA, I think that we should include steps to set it up. See [twofactorauth/notes/](https://github.com/2factorauth/twofactorauth/tree/master/notes)

The steps could be: 
1. [Login](https://groups.io/login) to your account. 
2. Click your email address in the top right-hand corner and click _account_. 
3. Click _security_. 
4. Click _set up two-factor authentication_. 
5. Enter your password and click _enable_. 
6. Follow the steps to enable 2FA. 

Or at least link to [this comment](https://github.com/2factorauth/twofactorauth/pull/4320#issuecomment-560177656).

**Review by Jawshy — DISMISSED** [review]
Would you kindly update the icon to use this smaller (file size) icon? 
groups_io

File: _data/communication.yml (hunk 1 above)
Comment (phallobst) [reply _data/communication.yml:147]: According to https://github.com/2factorauth/twofactorauth/pull/4199#issuecomment-546748150 the notes should not be necessary.

Comment (Carlgo11) [reply _data/communication.yml:147]: Yeah no need for documenting that.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**